earth_enterprise/src/fusion/tools/gee_test/status.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/python
+#!/bin/bash -eu
 #
-# Copyright 2017 Google Inc.
+# Copyright 2017 Open GEE Authors
```

Comment:
Minor: I've been saying "Open GEE Contributors".  We should pick one and stick with it.